### ISSUE TITLE
GFS.FR.MOGAL vs GFS.FR.MOGUL typo

### ISSUE THREAD

**Issue by tobigithub** [topepo/caret#296]
Hi,
there seems to be a name confusion betwenn GFS.FR.MOGAL  and GFS.FR.MOGUL which has been copied into caret. A search in the caret repo gives 2 results of "GFS.FR.MOGAL"

[GFS.FR.MOGUL in GitHub](https://github.com/search?utf8=%E2%9C%93&q=%22GFS.FR.MOGUL%22&type=Code&ref=searchresults)
[GFS.FR.MOGAL  in GitHub](https://github.com/search?utf8=%E2%9C%93&q=%22GFS.FR.MOGAL%22&type=Code&ref=searchresults)
[GFS.FR.MOGAL in caret] (https://github.com/topepo/caret/search?utf8=%E2%9C%93&q=GFS.FR.MOGAL)
 

Minimal test code (very very slow when use of correct spelling GFS.FR.MOGUL):
````R
require(caret); data(BloodBrain); t1 <- train(bbbDescr, logBBB, "GFS.FR.MOGAL"); t1;
# Error in train.default(bbbDescr, logBBB, "GFS.FR.MOGAL") : 
#  Model GFS.FR.MOGAL is not in caret's built-in library
````
Basically the short model name was spelled MOGAL and the model is names MOGUL
str(models$GFS.FR.MOGUL). Maybe this is already fixed in caret  caret_6.0-58

Cheers
Tobias

**Comment by topepo** [comment]
It is `GFS.FR.MOGUL` and the typo was fixed in the last CRAN version

**Comment by tobigithub** [comment]
Hi,
except for those two cases (not sure it matters that much):
https://github.com/topepo/caret/search?utf8=%E2%9C%93&q=MOGAL&type=Code
Cheers
Tobias
